spec/requests/hotel_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
```

Comment:
[OPTIONAL]
I think it'd be a good idea to change the file name to `hotels_controller_spec.rb` for consistency purposes.